charts/lenses/values.yaml

```diff
@@ -205,7 +205,7 @@ lenses:
   ## Schema registries should be behind a service, if so set one entry in the hosts
   ## If your schema registries are outside explicitly add them to the hosts.
   schemaRegistries:
-    enabled: true
+    enabled: false
```

Comment:
This smells like a breaking change and maybe should be handled as a major change. 

Use case:

- I install Chart version 3.2.3
- I have schemaRegistries and connectClusters running
- I evaluate the services
- I update to Chart version 3.2.4
- schemaRegistries and connectClusters are gone!
- I am confused

Reply:
the default values wrongly have enabled: true right? because with default values you are waiting to have already up and running schema registry, connect cluster. 

No need to be added though and most of people they are using their own values yaml right? I don't think that they use the chart just with the helm values.